cleancode.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
```

Comment:
I recommend passing the `-u` and `-e` flags to bash

Instead of `#!/bin/bash`, it should be `#!/bin/bash -ue`. In short, the `-u` flag "tells the shell to treat expanding an unset parameter an error, which helps to catch e.g. typos in variable names", while the `-e`flag "tells the shell to exit if a command exits with an error"

If you want to read more why [you can read this](https://www.gnu.org/software/bash/manual/html_node/The-Set-Builtin.html) or [this](https://unix.stackexchange.com/questions/595249/what-does-the-eu-mean-in-bin-bash-eu-at-the-top-of-a-bash-script-or-a) if you are having trouble sleeping 😂